### DISCUSSION TITLE
Botw on Poco x3 pro does not boot why?

### DISCUSSION THREAD

**Discussion by ARG14** [skyline-emu/skyline#1497] [General]
The Game boots on the menu already then why it doesn't on the poco x3 pro

**Comment by Allatel** [comment]
Check its version, maybe u have 1.4x or 1.5 or u are pirating xD. Devs said skyline wil support updates, so maybe u will be ok, but idk. U need save and 1.6.0 update. Save can be downloaded from discord, "check it!"

**Reply by ARG14** [reply]
What do you mean the game version?

**Reply by Allatel** [reply]
i mean game with all updates

**Reply by ARG14** [reply]
Well I think this is the base game because it weighs 13.4 GB and it seems to me that it is the normal one
